client/post-editor/editor-title/index.jsx

```diff
@@ -101,19 +104,36 @@ export default React.createClass( {
 				}
 				<TrackInputChanges onNewValue={ this.recordChangeStats }>
 					<TextareaAutosize
-						{ ...omit( this.props, Object.keys( this.constructor.propTypes ) ) }
+						{ ...ownProps }
 						className="editor-title__input"
-						placeholder={ this.translate( 'Title' ) }
+						placeholder={ translate( 'Title' ) }
 						onChange={ this.onChange }
 						onInput={ this.resizeAfterNewlineInput }
 						onBlur={ this.onBlur }
 						autoFocus={ isNew && ! isMobile() }
 						value={ post ? post.title : '' }
-						aria-label={ this.translate( 'Edit title' ) }
+						aria-label={ translate( 'Edit title' ) }
 						ref="titleInput"
 						rows="1" />
 				</TrackInputChanges>
 			</div>
 		);
 	}
-} );
+}
+
+export default connect(
+	( state, ownProps ) => {
+		const site = getSelectedSite( state );
+		const editedPostId = getEditorPostId( state );
+		const post = getEditedPost( state, site.ID, editedPostId );
+		const isNew = isEditorNewPost( state );
+
+		return {
+			isNew,
+			post,
+			site,
+			ownProps
```

Comment:
Yes I know, the idea is that ownProps needs to be passed down the textarea. The previous way of doing this was to rely on ownProps automatically merged to the props and using omit to exclude the current component props.

And when using localize and connect etc... we get extra props that we don't want to pass down to the textarea (we'll get warnings). And it's hard to keep those ignored props up to date with this method. So, for now, localize adds three props (translate, moment and another one). Imagine someone adding another prop to localize, he won't think of fixing these kind of issues (adding the new Prop to omit). That's why I used this technique.

I could also replace the ...ownProps with explicit props passing (we use only tabIndex for now I think), but I see the component as an upgraded textarea, so It's "logic" to just pass all parent props down.